Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -522,6 +522,37 @@ startActivity(i);
 
 The [Vulnerable App](https://github.com/clviper/android-fragment-injection/raw/master/vulnerableapp.apk "Vulnerable App Fragment Injection") and [Exploit PoC App](https://github.com/clviper/android-fragment-injection/blob/master/exploit.apk "PoC App to exploit Fragment Injection") are available for downloading.
 
+### Testing for Safe Browsing in WebViews (MSTG-PLATFORM-2)
```

Comment:
After writing all the other comments, I'm tending to think that if we're simply testing for SafeBrowsing as the Android SafeBrowsing feature, that wouldn't require a full section. However, if we're also including restrictions to a trusted domain things get more interesting and Android SafeBrowsing could be also included as an enhancement for a safer browsing experience. In that case, we might need another title for this section, I'm not sure yet.

Reply:
I think we can keep this section. It also talks about VirusTotal and we can extend it in the future with the point you raised about trusted domains. 
But we should change the title to "Testing URL Loading in WebViews (MSTG-PLATFORM-2)".